{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/layout.jinja2

```diff
@@ -58,7 +58,7 @@
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="//oss.maxcdn.com/libs/jquery/1.10.2/jquery.min.js" integrity="sha384-aBL3Lzi6c9LNDGvpHkZrrm3ZVsIwohDD7CDozL0pk8FwCrfmV7H9w8j3L7ikEv6h" crossorigin="anonymous"></script>
-    <script src="//oss.maxcdn.com/libs/twitter-bootstrap/3.0.3/js/bootstrap.min.js" integrity="sha384-s1ITto93iSMDxlp/79qhWHi+LsIi9Gx6yL+cOKDuymvihkfol83TYbLbOw+W/wv4" crossorigin="anonymous"></script>
+    <script src="//code.jquery.com/jquery-1.12.4.min.js" integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ=" crossorigin="anonymous"></script>
```

Comment:
One more instance of jquery, should be

`integrity="sha384-nvAa0+6Qg9clwYCGGPpDQLVpLNn0fRaROjHqs13t4Ggj3Ez50XnGQqc/r8MhnRDZ"`

Reply:
Hi think the jquery is fine. They are using sha256, you are using sha384 so the has will be different. But I got it from the CDN on jqueyr.com directory.

```html
script
	src="https://code.jquery.com/jquery-1.12.4.min.js"
	integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
	crossorigin="anonymous"></script>
		
The integrity and crossorigin attributes are used for Subresource Integrity (SRI) checking. This allows browsers to ensure that resources hosted on third-party servers have not been tampered with. Use of SRI is recommended as a best-practice, whenever libraries are loaded from a third-party source. Read more at srihash.org 
```

Reply:
I don't see the hash on https://code.jquery.com/ and the note you quoted says:

> Read more at [srihash.org](https://www.srihash.org/)

...which is from where I got the sha384 hash. Can you show where you got the sha256 hash? I can't verify it.

Reply:
Hi Steve,

Sure, you have to pull up the link for jquery 1 min. Here's the screenshot and code I included above.

![sha256](https://user-images.githubusercontent.com/2035561/34861781-a67d3942-f71b-11e7-87fc-929ee644a183.png)

Reply:
OK, and I forgot that I could just do this after downloading it:

`cat jquery-1.12.4.min.js | openssl dgst -sha256 -binary | openssl enc -base64 -A`

Double-verified!

Reply:
FTR, AdBlocker blocked that modal javascript for me, and I didn't notice it until just now.